projects/element-ng/search-bar/si-search-bar.component.ts

```diff
@@ -94,6 +95,16 @@ export class SiSearchBarComponent implements OnInit, OnDestroy, ControlValueAcce
   // eslint-disable-next-line @angular-eslint/no-input-rename
   readonly disabledInput = input(false, { alias: 'disabled', transform: booleanAttribute });
 
+  /**
+   * Aria label for the clear button.
+   *
+   * @defaultValue
+   * ```
+   * t(() => $localize`:@@SI_SEARCH_BAR.CLEAR_BUTTON:clear`)
+   * ```
+   */
+  readonly clearAriaLabel = input(t(() => $localize`:@@SI_SEARCH_BAR.CLEAR_BUTTON:clear`));
```

Comment:
```suggestion
  readonly clearButtonAriaLabel = input(t(() => $localize`:@@SI_SEARCH_BAR.CLEAR_BUTTON:clear`));
```

This makes it slightly more consistent